CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# dbt_hubspot version.version
+# dbt_hubspot v0.21.0
+[PR #154](https://github.com/fivetran/dbt_hubspot/pull/154) includes the following updates:
+
+## Breaking Changes
+- For Quickstart users, the models `hubspot__daily_ticket_history` and `hubspot__tickets` are now enabled by default.
```

Comment:
Small wording suggestion.

```suggestion
- For Quickstart users, the models `hubspot__daily_ticket_history` and `hubspot__tickets` are now available within the Quickstart UI and are enabled by default.
```

Also, let's add a note in here that if customers wish to turn this off then that can do so within the Data Models tab of the Quickstart Data Model job. This way customers are aware that they can disable this if they don't want this included in their job.